src/vmm/src/guest_config/x86_64/cpuid/amd/normalize.rs

```diff
@@ -292,18 +292,19 @@ impl super::AmdCpuid {
     ) -> Result<(), ExtendedCacheTopologyError> {
         for i in 0.. {
             if let Some(subleaf) = self.get_mut(&CpuidKey::subleaf(0x8000001d, i)) {
-                // Cache type. Identifies the type of cache.
+                // Cache level. Identifies the level of this cache. Note that the enumeration value
```

Comment:
nit: can add some scope in the commit message summary

Reply:
Is there a scope you'd suggest here?

Reply:
maybe `fix(vmm):` and mention `normalize` somehow?